synapse/federation/federation_client.py

```diff
@@ -357,7 +357,7 @@ def get_event_auth(self, destination, room_id, event_id):
         defer.returnValue(signed_auth)
 
     @defer.inlineCallbacks
-    def make_join(self, destinations, room_id, user_id, content):
+    def make_membership_event(self, destinations, room_id, user_id, membership, content):
```

Comment:
- Docstring?
- Can we assert that `membership in [Membership.Join, Membership.Leave]`?